File: Hydrogene/All/SharedUI/SharedUI.Shared/MainWindowController.Cocoa.cs
namespace SharedUI.Shared
{
	#if TOFFEE
	using AppKit;

	[IBObject]
	public partial class MainWindowController : NSWindowController
	{
		public this() : base withWindowNibName("MainWindow")
		{
			setup();
		}

		public override void windowDidLoad()
		{
			base.windowDidLoad();

			// Implement this method to handle any initialization after your window controller's
			// window has been loaded from its nib file.
		}

	}
	#endif
}

File: Iodine/All/SharedUI/SharedUI.Shared/MainWindowController.Cocoa.java
package SharedUI.Shared;

#if TOFFEE

@IBObject
public __partial class MainWindowController extends NSWindowController
{
	public MainWindowController()
	{
		super("MainWindow");
		setup();
	}

	@Override
	public void windowDidLoad()
	{
		super.windowDidLoad();
		//  Implement this method to handle any initialization after your window controller's
		//  window has been loaded from its nib file.
	}
}

#endif